src/nurse/models.py

```diff
@@ -23,10 +23,19 @@ def make_phone_field():
 class Patient(models.Model):
     firstname = models.CharField(max_length=30, blank=False)
     lastname = models.CharField(max_length=30, blank=False)
-    address = make_address_field()
+    street = make_street_field()
     zip_code = make_zip_code_field()
     city = make_city_field()
     phone = make_phone_field()
+    birthday = models.DateField(
+        auto_now=False, auto_now_add=False, blank=True, null=True
+    )
+    health_card_number = models.CharField(
+        max_length=15, blank=True, help_text="carte vitale"
+    )
+    ss_provider_code = models.CharField(
+        max_length=9, blank=True, help_text="code d’organisme de rattachement"
```

Comment:
I'm scared about anything that's not in the ascii table.
’ != '
It's my fault I left that dirty chart in my code suggestion sorry